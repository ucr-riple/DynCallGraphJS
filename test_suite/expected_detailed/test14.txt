{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test14.js:6:1:6:7)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test14.js:2:16:4:2)'
    ]
}