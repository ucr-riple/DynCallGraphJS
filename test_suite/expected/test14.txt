{
  '(test14.js:6:1:6:7)': [
    '(test14.js:2:16:4:2)'
  ]
}